modules/travel_pay/app/controllers/travel_pay/v0/claims_controller.rb

```diff
@@ -76,6 +78,22 @@ def appts_service
       def expense_service
         @expense_service ||= TravelPay::ExpensesService.new(auth_manager)
       end
+
+      def scrub_logs
```

Comment:
Is there a way to filter things [here instead](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/filter_parameter_logging.rb)?

Reply:
I looked at that, and thanks for bringing this up. 

We need to filter partial attributes only for certain routes (`show`), so I decided to do something more custom here.